### PR TITLE
[11.0][customer_outstanding_statement] minor fixes

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Applies similar fixes as in #516. 
Renames the report to 'Outstanding Statement'.

### CODE REVIEW

File: customer_outstanding_statement/report/customer_outstanding_statement.py [hunk 1]
```diff
@@ -94,8 +94,9 @@ def _display_lines_sql_q1(self, partners, date_end, account_type):
 
     def _display_lines_sql_q2(self):
         return """
-            SELECT partner_id, currency_id, move_id, date, date_maturity,
-                            debit, credit, name, ref, blocked, company_id,
+            SELECT Q1.partner_id, Q1.currency_id, Q1.move_id, 
+            Q1.date, Q1.date_maturity, Q1.debit, Q1.credit, 
+            Q1.name, Q1.ref, Q1.blocked, Q1.company_id,
```
Comment (MiquelRForgeFlow) [inline customer_outstanding_statement/report/customer_outstanding_statement.py:99]: I think this change is not needed, there is no joint, thus no problems. But if you want to proceed with this change, then you should also put the Q1 in the `currency_id` of the line below and fix the related flake8.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: customer_outstanding_statement/report/customer_outstanding_statement.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: Fixed.

Comment (JordiBForgeFlow) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: Fixed

Comment (MiquelRForgeFlow) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: still flake8 fails 🙁

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  Functional tests ok

File: customer_outstanding_statement/report/customer_outstanding_statement.py (hunk 1 above)
Comment (gdgellatly) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: why is it Q1 in a function called q2? am I just being stupid?

EDIT: Yes I am

Comment (gdgellatly) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: @mreficent IRT flake8, I have a more general problem with formatting SQL syntax to fit python style. Using triple quoted strings and laying it out nicely makes for easy reading of python code, but it generates truly horrible to read SQL logs. Honestly, I feel like we are better with flake8 overrides, but just a personal opinion.

But if you are going to lay it out for python, then you should indent continuation lines of major clauses (e.g. Q1.date and Q1.name should be indented) for readability.

Comment (MiquelRForgeFlow) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:99]: Previous flake8 error was that there was blank spaces and the end of some lines

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**